Utils/build-release-macos.sh

```diff
@@ -3,14 +3,10 @@ set -eux
 
 cd "$(dirname $0)/.."
 
-cmake -B Build/release-arm64 -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=On -DCMAKE_OSX_ARCHITECTURES=arm64 -DWITH_CLUB_FANTASTIC=On
-cmake --build Build/release-arm64 -j $(sysctl -n hw.logicalcpu)
-cmake --build Build/release-arm64 --target package
-cmake -B Build/release-arm64 -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=On -DCMAKE_OSX_ARCHITECTURES=arm64 -DWITH_CLUB_FANTASTIC=Off
-cmake --build Build/release-arm64 --target package
+cmake -B Build/release-universal -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=On -DCMAKE_OSX_ARCHITECTURES="arm64;x86_64" -DWITH_CLUB_FANTASTIC=On
```

Comment:
rootreducer found that this doesn't work for ffmpeg. I confirmed it in a github pipeline: https://github.com/florczakraf/itgmania/actions/runs/10355501236/job/28663288286#step:5:117

```
CMake Error at CMake/SetupFfmpeg.cmake:45 (message):
-- Configuring incomplete, errors occurred!
  Unsupported macOS architecture: arm64;x86_64, set CMAKE_OSX_ARCHITECTURES
  to either arm64 or x86_64
Call Stack (most recent call first):
  StepmaniaCore.cmake:212 (include)
  CMakeLists.txt:5 (include)
```

Reply:
Thanks for noting it,  fixed here: https://github.com/itgmania/itgmania/pull/423